src/content/en/2020/markup.md

```diff
@@ -452,7 +452,7 @@ Standard elements are those that are or were part of the HTML specification. Whi
   <figcaption>{{ figure_link(caption="Low probabilities of finding a given element in pages of the sample.", sheets_gid="184700688", sql_file="pages_element_count_by_device_and_element_type_present.sql") }}</figcaption>
 </figure>
 
-We're including these elements to give an idea what elements may have gone out of favor. But while `dir` and `basefont` were last specified in XHTML 1.0 (2000) and are no longer part of HTML, the rare use of `rp` (which was mentioned [as early as 1998](https://www.w3.org/TR/1998/WD-ruby-19981221/#a2-4) and is [still part of HTML](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-rp-element)), may just suggest that Ruby markup is not very popular.
+We're including these elements to give an idea what elements may have gone out of favor. But while `dir` and `basefont` were last specified in XHTML 1.0 (2000) and are no longer part of HTML, the rare use of `rp` (which was mentioned [as early as 1998](https://www.w3.org/TR/1998/WD-ruby-19981221/#a2-4) and is [still part of HTML](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-rp-element)), may just suggest that [ruby markup](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ruby) is not very popular.
```

Comment:
For consistency reasons with other references in the chapter I’d suggest to link to either https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-ruby-element (there’s no better section around the markup it seems) or, perhaps, https://www.w3.org/TR/ruby/, which is a somewhat historical but maybe for that reason interesting “base” document to learn about ruby markup.

Reply:
Any preference? I’m easy either way.

Reply:
Changed it to the second one. And also did the Dutch translation too.

Let me know if want to change to first, otherwise if happy to approve then will merge and include in next release.